FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/HealthCheckConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Configurations
+{
+    public class HealthCheckConfiguration
+    {
+        /// <summary>
+        /// The timeout to use for a single health check
+        /// </summary>
+        [JsonProperty("healthCheckTimeout")]
```

Comment:
Just out of curious here: is it the default behaviour of the .NET that it is using `NewtonSoft.Json` over `System.Text.Json` to parse configurations?